### PR TITLE
Update layouts.md to show label & title

### PR DESCRIPTION
**Pull request by tbenst**
I found this confusing that it was a array of arrays initially. Size looks a bit different than others on the page--not sure what settings were used? Let me know if you prefer issues instead of pull--I had to link to image in a fork of PlotReferenceImages.jl

### CODE REVIEW

File: docs/src/layouts.md [hunk 1]
```diff
@@ -39,7 +39,12 @@ More complex grid layouts can be created with the `grid(...)` constructor:
 plot(rand(100, 4), layout = grid(4, 1, heights=[0.1 ,0.4, 0.4, 0.1]))
 ```
 
----
+Titles and labels can be easily added:
+
+```@example layouts
+plot(rand(100,4), layout = 4, label=[["a"] ["b"] ["c"] ["d"]],
+    title=[["1"] ["2"] ["3"] ["4"]])
```
Comment (daschw) [inline docs/src/layouts.md]:
```julia
plot(rand(100, 4), layout = 4, label = ["a" "b" "c" "d"], title = ["1" "2" "3" "4"])
```
would achieve the same thing.

Comment (tbenst) [reply docs/src/layouts.md]: thanks!

Comment (tbenst) [reply docs/src/layouts.md]: I think this changed since I opened the pull-request, but indeed it works now

**Comment by tbenst** [comment]
Rebased & incorporated suggestion

File: docs/src/layouts.md [hunk 2]
```diff
@@ -39,6 +39,20 @@ More complex grid layouts can be created with the `grid(...)` constructor:
...
+
+
+Titles and labels can be easily added:
+
+```julia
+plot(rand(100,4), layout = 4, label=["a" "b" "c" "d"],
+    title=["1" "2" "3" "4"])
+```
+
+![](https://github.com/tbenst/PlotReferenceImages.jl/blob/master/PlotDocs/layout/layout_6.png)
```
Comment (daschw) [inline docs/src/layouts.md]: This image link also is not necessary.

File: docs/src/layouts.md [hunk 3]
```diff
@@ -39,6 +39,20 @@ More complex grid layouts can be created with the `grid(...)` constructor:
 plot(rand(100, 4), layout = grid(4, 1, heights=[0.1 ,0.4, 0.4, 0.1]))
 ```
 
+
+![](https://raw.githubusercontent.com/JuliaPlots/PlotReferenceImages.jl/master/PlotDocs/layout/layout_3.png)
+
+
+Titles and labels can be easily added:
+
+```julia
```
Comment (daschw) [inline docs/src/layouts.md]: replace this with
```@example layouts
like in the other code blocks.

File: docs/src/layouts.md [hunk 4]
```diff
@@ -39,6 +39,20 @@ More complex grid layouts can be created with the `grid(...)` constructor:
 plot(rand(100, 4), layout = grid(4, 1, heights=[0.1 ,0.4, 0.4, 0.1]))
 ```
 
+
+![](https://raw.githubusercontent.com/JuliaPlots/PlotReferenceImages.jl/master/PlotDocs/layout/layout_3.png)
+
```
Comment (daschw) [inline docs/src/layouts.md]: Thanks for the update! Sorry I forgot to mention another thing: Plots in the docs are now auto-generated. This means you can remove the image link here and below and instead ...

**Comment by daschw** [comment]
I added the reviews in the wrong order... should be read from third to first.

File: docs/src/layouts.md (hunk 3 above)
Comment (tbenst) [reply docs/src/layouts.md]: done

File: docs/src/layouts.md (hunk 4 above)
Comment (tbenst) [reply docs/src/layouts.md]: fancy! done.

**Comment by tbenst** [comment]
done! thanks

**Comment by daschw** [comment]
Thanks a lot and sorry you had to wait so long for this!

**Pull request merged**